pixelfusion/cavalry-api
# Feature/api prefix

| Q | A |
|--|--|
| Status | READY |
| Migrations | NO |
| Ticket | |

## Description
Removed the `api` part from the url, because its already in the subdomain.

## Steps to test/reproduce
1. pull the branch
2. run the tests `./vendor/bin/codecept run`
3. check the docs http://api.cavalry.local/documentation
